lib/process-services/src/lib/i18n/en.json

```diff
@@ -276,6 +276,7 @@
       "FORM": {
         "TITLE": "Start Process",
         "LABEL": {
+          "APPLICATIONS": "Select Application",
```

Comment:
Can you rename the key to SELECT_APPLICATION?

Reply:
Updated